doc/DEVELOPERS.md

```diff
@@ -64,9 +64,130 @@ make
 sudo make install
 ```
 
+### Configuration
+
+**If you have already set up the `myuser` and `repl` role, you can skip.**
+
+#### 1. Initialize PostgreSQL
+
+Open a new window, switch to the `postgres` user.
+
+``` sh
+sudo su -
+su - postgres
+```
+
+If you use dnf to install your postgresql, chances are the binary file is in `/usr/bin/`
```

Comment:
The YUM repository places binaries in `/usr/pgsql-15/bin`

Create a Fedora VM to verify